test/common/addresses.spec.ts

```diff
@@ -4,10 +4,16 @@ import { expect } from '../setup'
 import { getRandomAddress } from '../../src'
 
 describe('getRandomAddress', () => {
+  const random = global.Math.random
+
   beforeEach(async () => {
     global.Math.random = () => 0.5
   })
 
+  afterEach(async () => {
```

Comment:
Why `beforeEach` and `afterEach` over `before` and `after`?

Reply:
Yeah good point.

Reply:
Fixed.